claymore-frontend/src/components/Payload.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react";
```

Comment:
Do we actually need a payload employee page? It doesn't seem like important functionality and I don't think it was in our list of user stories. I'd advocate to remove this completely so we don't need to update/maintain it.

Reply:
I think we do need a page for employees to see customer orders, so we know who ordered what